File: conference/management-bc/web/src/main/java/com/microsoft/conference/management/domain/Events/SeatTypeRemoved.cs
using System;
using ENode.Eventing;

namespace ConferenceManagement
{
    public class SeatTypeRemoved : DomainEvent<Guid>
    {
        public Guid SeatTypeId { get; private set; }

        public SeatTypeRemoved() { }
        public SeatTypeRemoved(Guid seatTypeId)
        {
            SeatTypeId = seatTypeId;
        }
    }
}

File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/domain/event/SeatTypeRemoved.java
package com.microsoft.conference.management.domain.event;

import lombok.Getter;
import lombok.Setter;
import org.enodeframework.eventing.DomainEvent;

@Getter
@Setter
public class SeatTypeRemoved extends DomainEvent<String> {
    private String seatTypeId;

    public SeatTypeRemoved() {
    }

    public SeatTypeRemoved(String seatTypeId) {
        this.seatTypeId = seatTypeId;
    }
}
